doc-content/enterprise-only/openshift/apb-deploy-proc.adoc

```diff
@@ -39,6 +39,35 @@ Avoid each of the following combinations of settings. These combinations produce
 * *Red Hat - Single Sign-On* configured and  *Process server*>**Sets of Process Servers** exceeding `1`.
 ====
 +
+. If you are using the *External* database type for the {KIE_SERVER} in the *Authoring*, *Immutable Server - Process Server*, or *Managed Environment* option, set the parameters under the *External Database* heading. Set the host, port, database name, and database JDBC URL to the correct values for your database server. Use the following values for the other fields:
++
+** *Driver name*: The driver for the server, depending on the server type:
++
+*** mysql
+*** postgresql
+*** mariadb
+*** mssql
+*** db2
+*** oracle
+*** sybase
++ 
+** *Dialect class* (`KIE_SERVER_EXTERNALDB_DIALECT`): The Hibernate dialect for the server, depending on the server type:
++
+*** `org.hibernate.dialect.MySQL5Dialect` (used for MySQL and MariaDB)
+*** `org.hibernate.dialect.PostgreSQLDialect` 
+*** `org.hibernate.dialect.SQLServer2012Dialect` (used for MS SQL)
+*** `org.hibernate.dialect.DB2Dialect`
+*** `org.hibernate.dialect.Oracle12cDialect`
+*** `org.hibernate.dialect.SybaseASE15Dialect`
++
+If you created a custom image for using an external database server other than MySQL or PostgreSQL, as described in <<externaldb-build-proc>>, you must also set the *Process Server Image Stream Name* parameter (under the *Process Server* heading) to the following value:
```

Comment:
Seems like this bit "If you created a custom image ..." should either be step 8 or just be flush left (remove the + above it). Right now it's looks misaligned.